it/helpers/agent/ComponentSpecBase.scala

```diff
@@ -330,6 +330,11 @@ trait ComponentSpecBase extends TestSuite with CustomMatchers
       val formData = form.toFormMap.map { case (k, v) => (k -> Seq(v.getOrElse(""))) }
       post(s"/income-sources/add/uk-property-reporting-method?id=$testPropertyIncomeId", additionalCookies = additionalCookies)(formData)
     }
+
+    def postAddForeignPropertyReportingMethod(form: AddForeignPropertyReportingMethodForm)(additionalCookies: Map[String, String] = Map.empty): WSResponse = {
+      val formData = form.toFormMap.map { case (k, v) => (k -> Seq(v.getOrElse(""))) }
```

Comment:
maybe worth to raise and error instead of calling .getOrElse ?
I think in this context should be fine to call .get